packages/rest/src/__tests__/unit/coercion/utils.ts

```diff
@@ -68,11 +68,13 @@ export async function testCoercion<T>(config: TestArgs<T>) {
         pathParams.aparameter = config.rawValue;
         break;
       case 'query':
-        const q = qs.stringify(
-          {aparameter: config.rawValue},
-          {encodeValuesOnly: true},
-        );
-        url += `?${q}`;
+        {
```

Comment:
Weird, why are we wrapping this code in a `{}` block?

Can you find a different solution please, one that will require less code changes?

Reply:
I think it's due to https://eslint.org/docs/rules/no-case-declarations